roles/ydbd_dynamic/templates/ydbd-dynnode.service

```
@@ -13,13 +13,29 @@ PermissionsStartOnly=true
 Environment=LD_LIBRARY_PATH={{ ydb_dir }}/lib
 ExecStart={{ ydb_dir }}/bin/ydbd server \
     --yaml-config  {{ ydb_dir }}/cfg/ydbd-config-dynamic.yaml \
-    --grpcs-port {{ 2136 + item.offset }} --grpc-ca {{ ydb_dir }}/certs/ca.crt \
-    --ic-port {{ 19002 + item.offset }} --ca {{ ydb_dir }}/certs/ca.crt \
-    --mon-port {{ 8766 + item.offset }} --mon-cert {{ ydb_dir }}/certs/web.pem \
+{% if ydb_back_name is defined %}
+    --node-host {{ ydb_back_name }} \
+{% endif %}
+{% if ydb_front is defined %}
+    --grpc-public-host {{ ydb_front }} \
+{% endif %}
+{% if 'offset' in item %}
+    --grpcs-port {{ 2136 + item.offset }} \
+    --ic-port {{ 19002 + item.offset }} \
+    --mon-port {{ 8766 + item.offset }} \
+{% else %}
+    --grpcs-port 2136 --ic-port 19002 --mon-port 8766  \
+{% endif %}
     --tenant /{{ ydb_domain }}/{{ ydb_dbname }} \
 {% if ydb_request_client_certificate|bool %}
     --node-broker-use-tls true \
 {% endif %}
+{% if ydb_node_dc is defined %}
+    --data-center {{ ydb_node_dc }} --rack {{ ydb_node_rack }} --body {{ ydb_node_body }} \
+{% endif %}
```

Закрыл прошлый комментарий. Я имел ввиду поменять вот так: 
```suggestion
{% if ydb_node_dc is defined %}
    --data-center {{ ydb_node_dc }} \
{% endif %}
{% if ydb_node_rack is defined %}
    --rack {{ ydb_node_rack }} \
{% endif %}
{% if ydb_node_body is defined %}
    --body {{ ydb_node_body }} \
{% endif %}
```
Чтобы не было проблем, когда пользователь определил `ydb_node_dc`, но забыл `ydb_node_rack`. Или определил только `ydb_node_rack` и не понимает, почему его нет.
Раз это только в режиме совместимости, то названия можно не трогать